Lib/test/test_readline.py

```diff
@@ -413,6 +413,29 @@ def test_write_read_limited_history(self):
         # So, we've only tested that the read did not fail.
         # See TestHistoryManipulation for the full test.
 
+    @unittest.skipUnless(hasattr(readline, "set_pre_input_hook"),
+                         "set_pre_input_hook not available")
+    def test_get_pre_input_hook(self):
+        # Test that get_pre_input_hook returns None when no hook is set
+        get_pre_input_hook = getattr(readline, "get_pre_input_hook", None)
+        if get_pre_input_hook is None:
```

Comment:
I would prefer to check for `hasattr()` here, and use `readline.get_pre_input_hook()` in the code below.